### ISSUE TITLE
Better document SPA build target in the new guides

### ISSUE THREAD

**Issue by mornir** [nuxt/nuxtjs.org#902]
Hello!
I couldn't find any clear mention of the SPA feature in the new beta guides (which are great btw!).

For example, today I scaffolded a new project with the SPA option. Before there's was `mode: spa`, now there are `ssr: false` and `target: static`. I can understand the first one, but I always thought that `target: static` was for full static generation only.

Then I'm wondering which command I should use to build my project for production (as a SPA). Is it `nuxt generate` or `nuxt build`? I would have said `nuxt build`, but I also found in the old docs a spa flag used with the `generate` command.

### Bottom line
Nuxt is also an amazing tool for building SPAs. This feature should appear more prominently in the new guides 👍

**Comment by debs-obrien** [comment]
Hi @mornir thanks for the feedback. Actually we have removed the word SPA from the new docs because Nuxt is SPA by default since we have client side rendering no mater what the target is, server/static. So really what you need to decide is what your target is and if you want SSR, which is needed for server side rendered apps or static sites as it needs to pre-render the pages in order to have content or only client side rendering which in that case means no SSR (ssr:false). I suggest checking out these pages which explain it a bit better:

https://nuxtjs.org/guides/features/rendering-modes

https://nuxtjs.org/guides/features/deployment-targets

Also which command you use will depend on which target you are using as per stated in these docs:
https://nuxtjs.org/guides/get-started/commands

Hope that helps 😃

**Comment by mornir** [comment]
@debs-obrien Thanks for the detailed answer 😃 I think I was mainly confused because I was used to the old way of doing things _(where's my SPA mode?? 😄)_ And with vue-cli, it's the `build` command.

It makes sense to only keep one static mode. SPAs are actually also static. And in this regard, using `generate` is also the logical command.
